README.md

```diff
@@ -1,14 +1,51 @@
 # Getting Started with the .NET MAUI Tab View
-A quick-start project that helps you to integrate a Tab View control in a .NET MAUI application by adding the Syncfusion MAUI Tab View package from NuGet. This project contains simple code customizations, like how to add tabs items, populate the item source, and change the tab width mode, tab bar placement, and selection indicator placement.
-
-Syncfusion .NET MAUI Tab View control documentation: 
-https://help.syncfusion.com/maui/tabview/getting-started
-
-Check out these other MAUI Tab View examples on GitHub: 
-https://github.com/syncfusion/maui-demos/blob/master/MAUI/TabView/SampleBrowser.Maui.TabView/Samples/TabView/
+[.NET MAUI Tab View](https://www.syncfusion.com/maui-controls/maui-tab-view?utm_source=github&utm_medium=listing&utm_campaign=maui-tab-view-github-samples) is a simple and intuitive interface for tab navigation in mobile and desktop applications, where users can explore and switch between different tabs. A quick-start project that helps you to integrate a Tab View control in a .NET MAUI application by adding the Syncfusion MAUI Tab View package from NuGet. This project contains simple code customizations, like how to add tabs items, populate the item source, and change the tab width mode, tab bar placement, and selection indicator placement.
 
 ## Project prerequisites
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.

```

Comment:
Remove extra space here
![image](https://github.com/SyncfusionExamples/getting-started-with-the-.net-maui-tab-view/assets/95272441/cfdb278b-a7f1-41c4-8f6b-4cea0d293a0c)